torch/csrc/jit/serialization/import_export_constants.h

```diff
@@ -1,10 +1,13 @@
 #pragma once
+#include <string>
 
 namespace torch {
 namespace jit {
 constexpr size_t BYTECODE_INDEX_INSTRUCTION = 0;
 constexpr size_t BYTECODE_INDEX_OPERATOR = 1;
 constexpr size_t BYTECODE_INDEX_CONSTANT = 2;
 constexpr size_t BYTECODE_INDEX_TYPE = 3;
+constexpr char BYTECODE_VERSION_STR[] = "version";
+constexpr int64_t BYTECODE_VERSION_NUMBER = 1;
```

Comment:
do you expect to only support only one version at any point? Otherwise we can have Minimum/Producer/Maximum version similarly to how we do for main torchscript serialization

Reply:
@zdevito mentioned about Minimum/Producer/Maximum version in main container. Sounds a good idea to me and I'll follow up this route.